### PR TITLE
[refactor] update arceos dep version, refactor platform related config

### PR DESCRIPTION
**Pull request by hky1999**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +47,11 @@ First, you need to prepare your configuration file for the guest VM (several exa
 ```console
 $ cd arceos-vmm
 # x86_64
-$ make A=$(pwd) ACCEL=y BLK=y ARCH=x86_64 [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
-# aarch64 & riscv64
-$ make A=$(pwd) ACCEL=n BLK=y ARCH=[aarch64|riscv64] [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
+$ make ACCEL=y ARCH=x86_64 [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
+# aarch64
+$ make ACCEL=n ARCH=aarch64 [LOG=warn|info|debug|trace] PLATFORM=aarch64-qemu-virt-hv VM_CONFIGS=/PATH/TO/CONFIG/FILE run
```
Comment (aarkegz) [inline README.md:52]: Can we add a brief comment here to explain why `PLATFORM` is necessary here for aarch64?

File: arceos-vmm/Cargo.toml [hunk 2]
```diff
@@ -25,12 +25,13 @@ axstd = { workspace = true, features = [
 # System dependent modules provided by ArceOS-Hypervisor.
 axvm = { workspace = true }
 axvcpu = { workspace = true }
+axaddrspace = { workspace = true }
 
 # System independent crates provided by ArceOS, these crates could be imported by remote url. 
 axerrno = "0.1.0"
-memory_addr = "0.2.0"
+memory_addr = "0.3"
 page_table_entry = { version = "0.3.3", features = ["arm-el2"] }
```
Comment (aarkegz) [inline arceos-vmm/Cargo.toml:33]: Fix `page_table_entry` also please.

**Review by aarkegz — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -7,10 +7,10 @@ members = ["arceos-vmm"]
 lto = true
 
 [workspace.dependencies]
-axalloc = { git = "https://github.com/arceos-hypervisor/arceos.git", rev = "d938aca" }
-axhal = { git = "https://github.com/arceos-hypervisor/arceos.git", rev = "d938aca" }
-axstd = { git = "https://github.com/arceos-hypervisor/arceos.git", rev = "d938aca" }
-axtask = { git = "https://github.com/arceos-org/arceos.git", branch = "monolithic" }
+axalloc = { git = "https://github.com/arceos-hypervisor/arceos.git" }
```
[the remark this answers is not in the document]
Comment (hky1999) [reply Cargo.toml:10]: `vmm` is currently the default branch of arceos-hypervisor/arceos, but yes I think it‘s better to specify a branch here

File: README.md (hunk 1 above)
Comment (hky1999) [reply README.md:52]: As a matter of fact, I specify it in Makefile on my next PR.

File: arceos-vmm/Cargo.toml (hunk 2 above)
Comment (hky1999) [reply arceos-vmm/Cargo.toml:33]: ok

Comment (aarkegz) [reply arceos-vmm/Cargo.toml:33]: Also please check Cargo.lock to see if there are any crates with multiple versions referenced.

Comment (hky1999) [reply arceos-vmm/Cargo.toml]: > Also please check Cargo.lock to see if there are any crates with multiple versions referenced.

updated

**Review by aarkegz — APPROVED** [review]

**Pull request merged**